create_epicurve_date.R

```diff
@@ -0,0 +1,29 @@
+library(tidyverse)
+library(dotenv)
+library(REDCapR)
+library(lubridate)
+
+source("R/data_fetch_from_redcap.R")
+
+case_project_read <- get_case_records() %>% 
+  filter(redcap_event_name == "case_investigation_arm_1" &
+           is.na(redcap_repeat_instance)) %>% 
+  select(record_id, case_sx_onset_date, case_lab_collect_date, 
+         case_lab_report_date, case_report_datetime) %>% 
+  mutate_at(vars(case_sx_onset_date, case_lab_collect_date, 
+                 case_lab_report_date, case_report_datetime), as_date) 
+
+create_epicurve_date <- df %>% 
```

Comment:
I think you meant to say 
```suggestion
create_epicurve_date <- case_project_read %>% 
```

Everything makes sense when I make this substitution.